src/patternfly/components/EmptyState/docs/code.md

```diff
@@ -0,0 +1,16 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-hidden="true"` | `.pf-c-empty-state__icon` |  Hides icon for assistive technologies. **Required** |
+
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-empty-state` | `<div>` |  Initiates an empty state component. **Required** |
+| `.pf-c-empty-state__icon` | `*` |  Creates the empty state icon. |
```

Comment:
are you saying `.pf-c-empty-state_icon` can be applied to anything?

Reply:
Yeah, since it's used on span and other elements. Per https://fontawesome.com/v4.7.0/examples/

> You can place Font Awesome icons just about anywhere using the CSS Prefix fa and the icon's name. Font Awesome is designed to be used with inline elements (we like the <i> tag for brevity, but using a <span> is more semantically correct).

But looks like we use `<i>` everywhere else, so I'll just use `<i>` here. Good catch